db/seeds.rb

```diff
@@ -2,55 +2,62 @@
 
 require "faker"
 
-lambeth = LocalAuthority.find_or_create_by!(
```

Comment:
This is far too specific as some fields below may be different in your local db so just check if council code or subdomain exist before attempting to create it

Reply:
You can still use `find_or_create_by!` for this, by passing a block: the parameters to the method are the ones you search by (e.g., just `subdomain` perhaps), and then the block sets the remaining parameters.

Reply:
```ruby
lambeth = LocalAuthority.find_or_create_by!(subdomain: "lambeth") do |la|
   la.council_code = "LBH"
   la.short_name = "Lambeth"
   la.council_name = "Lambeth Council"
   la.applicants_url = "http://lambeth.bops-applicants.localhost:3001"
   la.signatory_name = "Christina Thompson"
   la.signatory_job_title = "Director of Finance & Property"
   la.enquiries_paragraph = "Planning, London Borough of Lambeth, PO Box 734, Winchester SO23 5DG"
   la.email_address = "planning@lambeth.gov.uk"
   la.feedback_email = "digitalplanning@lambeth.gov.uk"
   la.press_notice_email = "digitalplanning@lambeth.gov.uk"
 end

Reply:
Oh nice!